CalendarApp/NUnitTestProject/BookingServiceTests.DeleteTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3;
+using NUnit.Core;
+using NUnit.Framework;
+using BookingRoom.Helpers;
+using Microsoft.Practices.Unity;
+using GoogleCalendarService;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests : BaseIntegrationTest
+    {
+        private IUnityContainer _unityContainer;
+        private CalendarService _calendarService;
+        private BookingService _meetingBooking;
+
+        [TestFixtureSetUp]
+        public void Initialize()
+        {
+            _unityContainer = Dependency.UnityConfig.GetUnityContainer();
```

Comment:
I am not sure if private field is necessary, could it be transformed into local variable?